### PR TITLE
Add radix sort stepbystep, code templates, explanations

### LINKED ISSUE #12: Add Radix Sort code template(Bucket involved)

### LINKED ISSUE #89: Add radix sort stepbystep

### LINKED ISSUE #96: Improve colours of radix sort boxes

### PR DESCRIPTION
**Pull request by keanecjy**
* Also fixed some bugs in radix sort

Fixes #12 
Fixes #89 
Fixes #96

### CODE REVIEW

File: src/visualizer/codeinformation/explanations/radixSortEx.js [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
-    </span>
-  ),
-  stable: false,
-  inPlace: true,
+    'Radix sort works by sorting each digit from least significant digit to most significant ' +
+    "digit. So in base 10 (the decimal system), radix sort would sort by the digits in the 1's " +
+    'place, then the 10’s place, and so on. To do this, radix sort uses counting sort as a ' +
+    'subroutine to sort the digits in each place value.',
+  additionalDesc:
+    'd is the number of digits in the input numbers, b is the number of values each digit can have,',
```
Comment (seanjyjy) [inline src/visualizer/codeinformation/explanations/radixSortEx.js]: I think it's more accurate to represent b is the base for representing numbers?

**Review by seanjyjy — COMMENTED** [review]
LGTM 1 nit to fix

File: src/visualizer/codeinformation/explanations/radixSortEx.js (hunk 1 above)
Comment (keanecjy) [reply src/visualizer/codeinformation/explanations/radixSortEx.js]: Good point! Will fix it

**Comment by seanjyjy** [comment]
LGTM

**Pull request merged**